pkg/rootlessutil/xdg_linux.go

```diff
@@ -28,10 +29,11 @@ func XDGRuntimeDir() (string, error) {
 	}
 	// Fall back to "/run/user/<euid>".
 	// Note that We cannot rely on os.Geteuid() because we might be inside UserNS.
-	if euid := os.Getenv("ROOTLESSKIT_PARENT_EUID"); euid != "" {
-		return "/run/user/" + euid, nil
```

Comment:
If you want to validate the format of `$ROOTLESSKIT_PARENT_EUID`, a validation code can be added here
```go
if _, err := strconv.Atoi(euid); err != nil {
  return fmt.Errorf("invalid $ROOTLESSKIT_PARENT_EUID value: %q: %w", euid, err)
}
```